web/src/pages/api/user/confirmEmail.ts

```diff
@@ -1,23 +1,35 @@
 import { User } from "@prisma/client";
 import { NextApiRequest, NextApiResponse } from "next";
 
-import { prisma } from "../../../../server/db/client";
-import { validateMethod } from "../../../../utils/validation";
+import { prisma } from "../../../server/db/client";
+import { validateJWT } from "../../../utils/jwt";
+import { validateMethod } from "../../../utils/validation";
 
 export default async function confirmEmail(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
   if (!validateMethod("GET", req, res)) return;
 
-  const { userID } = req.query;
+  const userID = req.query.id;

```

Comment:
To keep it consistent the other endpoints let's name it `userId` with lowercase d